website/addons/wiki/model.py

```diff
@@ -51,7 +54,39 @@ def subscribe_on_write_permissions_revoked(node):
         wiki_utils.migrate_uuid(node, wiki_name)
 
 
-def build_wiki_url(node, label, base, end):
+# subscribes to project model wiki update
+@wiki_updated.connect
+def subscribe_wiki_updates(sender, node, user, version, page_name, rename=None, deleted=None):
```

Comment:
I lean toward leaving this the way it is, unless this function becomes too complex (in that case it may be more appropriate to call helper functions from some place like website/addons/wiki/utils).

Consider changing the signature to 
```def subscribe_wiki_updates(sender, node, user, version, page_name, action='changed')```
rather than a series of kwargs (only one of which should ever be Truthy)